app/views_redesigned/devise/passwords/new.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render "devise/layout" do %>
+<%# <%= render "devise/layout" do %1> %>
```

Comment:
If we aren't using this, go ahead and remove the commented out code.

Reply:
Nice :+1: